src/components/Payment/index.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import creditcard from '../../assets/creditcard.png';
+import date from '../../assets/date.png';
+import coin from '../../assets/coins.png';
+import locked from '../../assets/locked.png';
+
+import { Input } from 'reactstrap';
+
+
+class Payment extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      cardNumber: '',
+      noSpaceCardNumber: '',
+      expiryDate: '',
+      amount: '',
+      amountWithoutComma: '',
+      cvs: ''
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+
+  handleChange(e) {
+    let val = e.target.value.replace(/\D/g, '');
+    switch (e.target.id) {
+      case 'cardNumber':
+        let addSpace = val.replace(/(.{4})/g, '$1 ').trim();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected lexical declaration in case block.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)